### PR TITLE
Release 1.0.13 and fix bug when push alert is an object

### PR DESCRIPTION
**Pull request by drew-gross**

### CODE REVIEW

**Comment by facebook-github-bot-5** [comment]
By analyzing the blame information on this pull request, we identified @JeremyPlease, @flovilmart and @durunvo to be potential reviewers.

File: src/dashboard/Push/PushIndex.react.js [hunk 1]
```diff
@@ -145,7 +145,7 @@ let getPushName = (pushData) => {
       payload = JSON.parse(payload);
     } catch(e) { }
     if(payload){
-      return payload.alert ? payload.alert : payload;
+      return payload.alert ? payload.alert.toString() : payload;
```
Comment (JeremyPlease) [inline src/dashboard/Push/PushIndex.react.js]: @drew-gross If `payload.alert` is an object then the push name will be  "[object Object]". Can we do the following instead to handle objects and strings:

```
return payload.alert ? JSON.stringify(payload.alert) : payload;
```

**Comment by JeremyPlease** [comment]
One comment/suggestion. Otherwise, looks good!

**Comment by drew-gross** [comment]
Yeah thats definitely a better idea. Thanks!

**Comment by facebook-github-bot-8** [comment]
@drew-gross updated the pull request.

**Pull request merged**